Tools/sitl_run.sh

```diff
@@ -99,10 +99,19 @@ for file in "$@"; do
 done
 
 export PX4_SIM_MODEL=${model}
+export PX4_SIM_PROGRAM=${program}
 
 SIM_PID=0
 
-if [ "$program" == "jmavsim" ] && [ ! -n "$no_sim" ]; then
+if [ "$program" == "sih" ] && [ ! -n "$no_sim" ] && [[ ! -n "$HEADLESS" ]]; then
+	# Start Java simulator for displaying sih
+	if [ "$model" == "plane" ]; then
+		"$src_path"/Tools/jmavsim_run.sh -r 250 -o -a &
+	else
```

Comment:
This would result in every model being executed as the same multirotor model. I think this would potentially lead to confusion when the target `make px4_sitl_nolockstep sih_rover` being supported and doesn't fail

Reply:
I changed it to an elif